core/primitives/src/errors.rs

```diff
@@ -647,6 +637,10 @@ pub struct BalanceMismatchError {
     pub processed_delayed_receipts_balance: Balance,
     #[serde(with = "dec_format")]
     pub initial_postponed_receipts_balance: Balance,
+    // TODO(congestion_control): remove cfg on stabilization
+    #[cfg(feature = "nightly")]
```

Comment:
Would it make sense to add those fields without the feature gating? They should be set to zero before cc stabilization anyway.

Reply:
I'd rather not have the field show up in RPC nodes and explorers until we stabilize the feature.